### PR TITLE
Remove leakage of sensitive information

### PR DESCRIPTION
**Pull request by harture**
There is some space for improvements and we will have to fix this issue in all our modules

### CODE REVIEW

File: cmd/keycloakb/keycloak_bridge.go [hunk 1]
```diff
@@ -831,7 +841,11 @@ func config(logger log.Logger) *viper.Viper {
 	sort.Strings(keys)
 
 	for _, k := range keys {
-		logger.Log(k, v.Get(k))
+		if _, censored := censoredParameters[k]; censored {
```
Comment (fperot74) [inline cmd/keycloakb/keycloak_bridge.go]: I would add another condition : if "password" is found in the parameter name, it is automatically censored (and perhaps some other keywords)

Comment (harture) [reply cmd/keycloakb/keycloak_bridge.go]: Added

Comment (fperot74) [reply cmd/keycloakb/keycloak_bridge.go]: With your latest fix, you don't need anymore rows like `censoredParameters["XXX-password"] = true`

Comment (fperot74) [reply cmd/keycloakb/keycloak_bridge.go]: Another idea: only 2 branches for if statement with 

> censored || strings.Contains(k, "password")

**Comment by harture** [comment]
(Test pour voir si ça marche de répondre dans les mails.)
Je vois le fix comme un fallback mais je penses que on doit continuer a le faire explicitement.

**Pull request merged**